apps/docs/src/content/reference/extras/use-fbo.mdx

```diff
@@ -0,0 +1,28 @@
+---
+order: 14.55
+category: '@threlte/extras'
+sourcePath: 'packages/extras/src/lib/hooks/useCursor.ts'
+name: useFBO
+type: 'hook'
+---
+
+useFBO (Framebuffer Object) is a hook that creates a `THREE.WebGLRenderTarget`. This is a port of [drei's useFBO hook](https://github.com/pmndrs/drei#usefbo).
```

Comment:
I'd love to learn more about this. It seems like one of those things that's super powerful but not talked about.

I'll have a look around on three.js and drei to see if there's some more detail / guidance / tutorial we could offer here in our docs.

if you'd like to, perhaps tell us about how this came up for you?

Reply:
quick references for this:
[three.js docs](https://threejs.org/docs/index.html?q=render%20target#api/en/renderers/WebGLRenderTarget)
[drei example 1](https://drei.pmnd.rs/?path=/story/misc-usefbo--use-fbo-st)

Reply:
I added a couple lines explaining frame buffers with a high level example. I plan on contributing some material ports from the drei separately that depend on this hook which could be referenced in the future as living examples.

Reply:
very nice! thank you